sql/2021/privacy/trackers_whotracksme.py

```diff
@@ -0,0 +1,44 @@
+"""
+Retrieve and extract trackers as identified by WhoTracks.me.
+
+First, download the latest 'tracker database'
+(https://github.com/ghostery/whotracks.me/tree/master/whotracksme/data/assets):
+`aws s3 cp --no-sign-request s3://data.whotracks.me/trackerdb.sql .`
+
+Then, execute this script.
+Finally, copy the result into the tracker-related SQL queries.
+"""
```

Comment:
Let's get the results of this query added as a date table to almanac schema. We have precedence in this with other chapters (e.g. `h2_prioritization_cdns` is a special table just for the HTTP chapter from an external source).

That will make your other queries in this PR a lot smaller as not repeating the whole dataset each time.

Will also make it easier to manage.

Can you reach out to @paulcalvano to get this set up?
Should also consider this for the haveIbeenpwned data.

BTW are there any licencing issues with using either o these datasets?

Reply:
HaveIBeenPwned is on CC licence - https://haveibeenpwned.com/API/v3#License
and whotracks me is on MIT - https://github.com/ghostery/whotracks.me/blob/master/LICENSE.md
So I suppose there is no issue with licensing, right?

@paulcalvano how would you prefer us to prepare this data for BigQuery?

Reply:
Licencing looks good to me!

If you provide the data in a Google Sheets document I imagine @paulcalvano can get that inserted for you!

Reply:
@paulcalvano could you upload [this CSV](https://github.com/HTTPArchive/almanac.httparchive.org/blob/privacy-sql-2021/sql/2021/privacy/trackers.csv) to the BigQuery please, so that we can relate to it in the queries [for example](https://github.com/HTTPArchive/almanac.httparchive.org/pull/2227/files#diff-99583fbced3f94f4f9705727dfc2f600d12225cb24cb7c1a2351ad6c44046679R4)?